### PR TITLE
bumping version to 1.2.0, editing changelog

### PR DESCRIPTION
**Pull request by grawk**

### CODE REVIEW

**Comment by aredridel** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+##### v.1.2.0
```
Comment (thefourtheye) [inline CHANGELOG.md]: Minor nit. None of the other versions have `v.`

Comment (aredridel) [inline CHANGELOG.md]: Oh well spotted.

Comment (thefourtheye) [inline CHANGELOG.md]: Oh, I didn't know that. (Not a native English speaker :D) Thanks :-)

**Pull request merged**